redaxo/src/addons/phpmailer/lib/cronjob.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * PHPMailer Addon.
+ *
+ * @author markus[dot]staab[at]redaxo[dot]de Markus Staab
+ *
+ * @package redaxo\phpmailer
+ */
+
```

Comment:
```suggestion
/**
 * @package redaxo\phpmailer
 */
```

Wir haben solche Kommentare teils noch drin, neu würde ich die so aber nicht mehr verwenden.